app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/itemsList/itemsListHeader/itemsListHeader.scss

```diff
@@ -31,6 +31,10 @@
 
 .switcher {
   width: auto;
+
+  &.shown-less {
```

Comment:
It's better to use conditional rendering instead of `display` style. Just render switcher in case when it needed.